platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java

```diff
@@ -268,6 +270,7 @@ public synchronized static void initMetricsRegistry(String serviceName, Config c
     new JvmThreadMetrics(DEFAULT_TAGS).bindTo(METER_REGISTRY);
     new JvmMemoryMetrics(DEFAULT_TAGS).bindTo(METER_REGISTRY);
     new UptimeMetrics(DEFAULT_TAGS).bindTo(METER_REGISTRY);
+    new Log4j2Metrics(DEFAULT_TAGS).bindTo(METER_REGISTRY);
```

Comment:
which ones are we interested in ? error rate metric ?

Reply:
want to monitor and add alert on overall log volume which includes all levels (debug, info, warn, error, fatal).
we can have a different alert at error/warn rate as well.